pkg/config/spec.go

```diff
@@ -63,9 +63,17 @@ func NewInstallSpec(cfg *Config) (*v1.InstallSpec, error) {
 	activeImg.FS = constants.LinuxImgFs
 	activeImg.MountPoint = constants.ActiveDir
 
+	// First try to use the install media source
 	if isoRootExists {
 		activeImg.Source = v1.NewDirSrc(constants.IsoBaseTree)
-	} else {
+	}
+	// Then any user provided source
+	if cfg.Install.Source != "" {
```

Comment:
Shouldn't this be tried before `isoRootExists` ? If the user explicitly specified a source image, shouldn't we just use that?

Reply:
it goes in descending order and overwrites it if found to maintain what we used to did.

So:
 - set to isobasetree if it exists
 - overwrite with user provided source if user provided it

This way, if we dont provide the source it still falls back to the isotree as it did before
And then if neither of those are available it creates an empty source as it did before.

If we did the other way, the isorootree woul overwrite the user provided source and we dont want that.

Reply:
oh ok, I didn't expand all the code. The order just seemed wrong. All good.